02-express-tutorial/answers/controllers/people.js

```diff
@@ -0,0 +1,58 @@
+let { people } = require('../../data');
+
+const getPeople = (req, res) => {
+	res.status(200).json({ success: true, data: people });
+};
+
+const createPerson = (req, res) => {
+	const { name } = req.body;
+	!name
+		? res.status(400).json({ success: false, msg: 'please provide name value' })
+		: null;
+	res.status(201).json({ success: true, person: name });
+};
+
+const createPersonPostman = (req, res) => {
+	const { name } = req.body;
+	!name
+		? res.status(400).json({ success: false, msg: 'please provide name value' })
+		: null;
+	res.status(201).json({ success: true, data: [...people, name] });
```

Comment:
Same as comment above about ternary operator

Reply:
Correnction made. Thank you.